crates/oxc_minifier/src/printer/gen.rs

```diff
@@ -122,6 +140,36 @@ fn print_if(if_stmt: &IfStatement<'_>, p: &mut Printer) {
     }
 }
 
+// <https://github.com/evanw/esbuild/blob/e6a8169c3a574f4c67d4cdd5f31a938b53eb7421/internal/js_printer/js_printer.go#L3444>
+fn wrap_to_avoid_ambiguous_else(stmt: &Statement) -> bool {
+    let mut current = stmt;
+    loop {
+        current = match current {
+            Statement::IfStatement(Box(IfStatement { alternate, .. })) => {
+                if let Some(stmt) = &alternate {
+                    stmt
+                } else {
+                    return true;
+                }
```

Comment:
Can this be 1 line like `wrap_or(true)`?

Reply:
The `Some` branch is an expression return, the else branch is a statement return (exits the function body). So this can't be reduced, clippy will shout at us if it can be reduce.